### DISCUSSION TITLE
Is there a way to reverse the "entrify/categories" command?

### DISCUSSION THREAD

**Discussion by adrianjean** [craftcms/cms#15162] [Q&A] [answered]
I have a CraftCM 4.x site that I was eager to jump on the entrification train in anticipation of CraftCMS 5.x

So I ran `entrify/categories` on all of my categories and 'entrify/global-set' for my global sets.

However, now that CraftCMS 5 is ready, I'd like to upgrade my site to CraftCMS 5 -- but before I do that I'd like to convert my categories back to traditional categories so that everything is organized again and do the upgrade. I noticed that Categories and Global sets are still in CraftCMS and I could really use the organization again. 

**Question 1:** 
Is there a way to convert the categories back to traditional categories in CraftCMS 4.x? (Restoring from backup is not an option as I have been running the site now for over a year since converting)

**Question 2:**
If there is no way to re-convert back to traditional categories, can I upgrade to CraftCMS 5, and then some how flag that certain sections are meant to be categories?

Right now my entries page has so many entry types (sections, categories, global sets), it's quite intense for users to manage the site.

Help!?

**Comment by brandonkelly** [comment, marked as answer]
There’s no automated way to reverse it, sorry. We are still playing with some ideas to improve the organization of your content, which should help, though.

**Comment by brandonkelly** [comment]
Craft 5.9.0 is out with the ability to create custom entry index pages (#17779) 🎉
